src/gax/src/polling_error_policy.rs

```diff
@@ -59,6 +59,7 @@ pub trait PollingErrorPolicy: Send + Sync + std::fmt::Debug {
     ///   attempt. This method called after LRO successfully starts, it is
     ///   always non-zero.
     /// * `error` - the last error when attempting the request.
+    #[cfg_attr(not(feature = "_internal-semver"), doc(hidden))]
```

Comment:
comment: Hm, I was thinking that if we don't use these downstream, semver should be unaffected.

But I know the polling policy, at least, is exposed in stubs with LROs.